lib/game.rb

```diff
@@ -77,11 +71,20 @@ def cruiser_chooser
       end
     end
     @player_board.place(@p_cruiser, cords)
-    puts "==============PLAYER BOARD==============\n"
-    print @player_board.render(true)
+    player_board_output
     sub_chooser
   end
   
+  def player_board_output
+    puts "==============PLAYER BOARD==============\n"
+    print @player_board.render(true)
+  end
+
+  def computer_board_output
```

Comment:
good call on these also we use them alot and it looks nice having it as a method